backend/config.js

```diff
@@ -40,7 +40,8 @@ const configData = envSchema({
     .prop("S3_CDN_BUCKET", S.string().default("fp-dev-cdn"))
     .prop("SENTRY_DSN", S.string())
     .prop("SENDGRID_API_KEY", S.string())
-    .prop("SENDGRID_CONTACTS_LIST_ID", S.string()),
+    .prop("SENDGRID_CONTACTS_LIST_ID", S.string())
+    .prop("UNSUBSCRIBE_KEY", S.string()),
```

Comment:
can we use the same `AUTH_SECRET_KEY` we already have? if we did, we can get rid of the new "jwt-simple" dependency, and just use the existing `app.jwt.decode(token)`

Reply:
@mannykary also suggested this on slack, I didn't find how to encode with "Fastify-jwt" at that time so I proposed to use "jwt-simple" instead, I will double-check this, (it seems the `sign` method can be used here instead)

Reply:
yes the `sign` method is using for encoding, please check [this small example ](https://github.com/fastify/fastify-jwt#example-3)

Reply:
code updated with package `jsonwebtoken`, tested with Postman and F/E

Reply:
@mannykary the only change at Lambda side will be changing the key to be `AUTH_SECRET_KEY`